constantine/serialization/codecs_banderwagon.nim

```diff
@@ -44,6 +46,27 @@ func validate_scalar*(scalar: matchingOrderBigInt(Banderwagon)): CttCodecScalarS
     return cttCodecScalar_ScalarLargerThanCurveOrder
   return cttCodecScalar_Success
 
+func make_scalar_mod_order*(reduced_scalar: var Fr[Banderwagon], src: array[32, byte], order: static Endianness = bigEndian): bool =
+  ## Convert a 32-bytes digest to a Verkle Trie protocol field element
```

Comment:
The comment below was for fromVerkleDigest, now that doc comment looks out-of-place

```suggestion
  ## Convert a 32-byte array to a field element, reducing it modulo Banderwagon's curve order if necessary.
```